examples/vite/src/index.scss

```diff
@@ -30,6 +30,10 @@ body,
     flex-shrink: 0;
     box-shadow: 0 0 8px rgba(0, 0, 0, 0.15);
 
+    .str-chat__channel-preview-messenger {
+      max-width: 1000px;
```

Comment:
What if we added the max-width on the ChanneList root in this file and remove the channel preview CSS?

Reply:
Sure - will try it out and see